src/applications/avs/containers/Avs.jsx

```diff
@@ -112,8 +117,6 @@ const Avs = props => {
             <MoreInformation avs={avs} />
           </va-accordion-item>
         </va-accordion>
-
-        <Footer avs={avs} />
```

Comment:
Just to confirm, this was supposed to be delete

Reply:
yep, thanks for checking! the new header has the generation date, so we took it out to avoid duplication.